paket.lock

```diff
@@ -1,156 +1,156 @@
 REDIRECTS: ON
-RESTRICTION: || (== net45) (== net452) (== netcoreapp2.1) (== netstandard2.0)
+RESTRICTION: || (== net20) (== net40) (== net45) (== net452) (== netcoreapp2.1) (== netstandard2.0)
```

Comment:
looks like you had or have some .net 2 and .net 4 projects at the time of lock creation - try to stay at net45 or 452 where necessary

Reply:
It was a missed Mono project. I have removed it as it should not be used anymore.